packages/medusa/src/utils/__tests__/transaction/transaction-builder.ts

```diff
@@ -0,0 +1,376 @@
+import e from "cors"
+import { TransactionBuilder } from "../../transaction/transaction-builder"
+
+describe("TransactionBuilder", () => {
+  describe("Basic methods", () => {
+    let transactionBuilder: TransactionBuilder
+
+    beforeEach(() => {
+      transactionBuilder = new TransactionBuilder()
+    })
+
+    test("load", () => {
```

Comment:
todo: `should load and build the actions` something more descriptive, can I let you update the other ones as well. I am trying to push for some more convention around the tests :)